plugins/inputs/lotus/lotus.go

```diff
@@ -26,6 +29,12 @@ const (
 	DefaultConfig_ChainWalkThrottle = "1s"
 )
 
+const (
+	MpoolAdd       = api.MpoolAdd
+	MpoolRemove    = api.MpoolRemove
+	MpoolBootstrap = MpoolRemove + 1
```

Comment:
Instead of adding a new column as https://github.com/filecoin-project/sentinel/issues/36 suggests I have extended the update types here, this alright with you @placer14?

Reply:
That sounds good to me. Maybe Bootstrap shouldn't be `+1` though in case another type gets added to that iota in lotus. How about `-1`?